cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -535,6 +535,10 @@ To find out the address of an program's container host and the container’s deb
 CDAP for a flow or service’s live info via an HTTP GET method::
 
   GET <base-url>/namespaces/<namespace>/apps/<app-id>/<program-type>/<program-id>/live-info
+  
+If you are trying to debug a system service, this endpoint gives the container info for a system service::
```

Comment:
this information should be a section in http://docs.cdap.io/cdap/current/en/reference-manual/http-restful-api/monitor.html where the other information about system services lives.  System services are CDAP services, which should not get mixed up with 'service' programs that users write.